sark/code/line.py

```diff
@@ -180,9 +180,19 @@ def endEA(self):
     def disasm(self):
         """Line Disassembly"""
         return idc.GetDisasm(self.ea)
+    @property
+    def type(self):
+        """return the type of the Line """
+        properties = {self.is_code:"code",
+                      self.is_data:"data",
+                      self.is_string:"string",
+                      self.is_tail:"tail",
+                      self.is_unknown:"unknown"}
+        for k, v in properties.items():
+            if k: return v
 
     def __repr__(self):
-        return "[{:08X}]    {}".format(self.ea, self.disasm)
+        return "[{:08X}]    {} {}".format(self.ea, self.disasm, self.type)
```

Comment:
I added it when debugging a script that dealt with data ,and ignored unknowns. wanted to be able to tell what is what at a glance 
for example in cases like this 
`DCB 0x33 ; 3 `
this was marked as unknown , I'd assume it's data 
if you think it's unnecessary in most cases I can remove it